spec/features/event_disavowal_spec.rb

```diff
@@ -22,14 +22,20 @@
     disavow_last_action_and_reset_password
   end
 
-  scenario 'disavowing a new device sign in' do
-    allow(IdentityConfig.store).to receive(:otp_delivery_blocklist_maxretry).and_return(3)
-    signin(user.email, user.password)
-    Capybara.reset_session!
-    visit root_path
-    signin(user.email, user.password)
-
-    disavow_last_action_and_reset_password
+  context 'when aggregated new device alerts is disabled' do
```

Comment:
Can we add test case(s) for the opposite context "when aggregated new device alerts is enabled"

Reply:
Added that test. It concludes by checking a link in an email that doesn't ever get sent while the feature flag is set in this branch so I removed that function from the opposite context test.